dask/dataframe/io/parquet.py

```diff
@@ -171,3 +183,8 @@ def to_parquet(path, df, encoding=default_encoding, compression=None,
 
     with myopen(sep.join([path, '_common_metadata']), mode='wb') as f:
         fastparquet.writer.write_common_metadata(f, fmd)
+
+
+@partial(normalize_token.register, fastparquet.ParquetFile)
```

Comment:
Makes `dask.base.tokenize(my_parquetfile)` return a sensible result, which in turn helps our names to be deterministic and non-conflicting.